src/startStopScene.ts

```diff
@@ -69,6 +69,27 @@ export async function startScene(lineStart?: number) {
         lineNumber--;
     }
 
+    // If current line contains: "def construct(" without "#" before it,
+    // then we insert a special comment line, and move the cursor to it
+    // (because `manimgl -se <lineNumber>` doesn't work on the "def construct(" line)
+    let idx = lines[lineNumber].indexOf("def construct(");
+    if (idx !== -1) {
+        if (lines[lineNumber].slice(0, idx).includes("#")) {
+            idx = -1;
+        }
+    }
+    if (idx !== -1) {
+        lineNumber++;
```

Comment:
Instead of this, write one single if-statement: `if (idx !== -1 && !lines[lineNumber].slice(0, idx).includes("#"))`